### DISCUSSION TITLE
MariaDB dump: start use mariadb-dump not mysqldump

### DISCUSSION THREAD

**Discussion by meriturva** [dbeaver/dbeaver#20198] [Q&A]
### Description

I would like to add an argument during dump to disable "slow query log" so on `mariadb-dump` is present on recent versions `--skip-log-queries`.

Now it is impossible to use that flag due to the use of original `mysqldump` command.

### DBeaver Version

Community editor 23.0.5

### Operating System

Windows 11

### Database and driver

MariaDb 3.0.7

**Comment by LonwoLonwo** [comment]
Hello @meriturva 

Sorry, I am a little confused.

You can choose MariaDB dump (if you have it on your PC):

![2023-06-01 12_04_24-Window](https://github.com/dbeaver/dbeaver/assets/45152336/9a695db8-34c2-4c88-9d9f-9d5c0331b31d)

You can use custom arguments:

![2023-06-01 12_05_59-Window](https://github.com/dbeaver/dbeaver/assets/45152336/f1c86458-ac7c-44e9-a131-d62fd6f65a6d)

![2023-06-01 12_05_41-Window](https://github.com/dbeaver/dbeaver/assets/45152336/16751e83-508a-4505-aa95-001e7e43c1dd)

So, what is the question?

**Comment by meriturva** [comment]
That's something I didn't know before your answer.

Great and thanks for your time!
